app/javascript/spec/getJSONFixtures.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+const fixturesPath = './fixtures/json';
```

Comment:
can you also move this file in `spec/helpers/` please? (so it will be `../fixtures/json`)

just so that we don't mix specs and helpers for those specs